_adoc/build.sh

```diff
@@ -73,7 +74,7 @@ do
 	if [[ -n "$frontmatter" ]]
 	then
 		echo "---"
-		echo "$frontmatter"
+		echo -e "$frontmatter"
```

Comment:
What does -e do?  It isn't defined on macOS at least....

Reply:
-e enables interpretation of backslash-escape sequences, like the `\n` in the frontmatter. Under linux/bash, without this flag, the frontmatter results in such a line:

````
---
version: 1.1.2\nlayout: default
---
````

If the 'echo' command behaves differently on macos, maybe the solution is to keep the frontmatter in 2 variables and 'echo' them one after the other to avoid the `\n` escaping problem.